Spanner/composer.json

```
@@ -7,7 +7,7 @@
         "php": "^8.0",
         "ext-grpc": "*",
         "google/cloud-core": "^1.52.7",
-        "google/gax": "^1.30"
+        "google/gax": "dev-main as 1.32"
```

This will need to be `^1.32` once we merge https://github.com/googleapis/gax-php/pull/562 

```suggestion
        "google/gax": "^1.32"
```